src/scss/custom.scss

```diff
@@ -59,10 +62,123 @@
   }
 }
 
+// Side Bar
+.sidebar-section {
+  color: var(--doc-sidebar-section-title);
+  padding: var(--ifm-menu-link-padding-vertical) var(--ifm-menu-link-padding-horizontal);
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.sidebar-section:not(:first-child) {
+  margin-top: calc(var(--ifm-menu-link-padding-vertical) * 2);
+}
+
+.menu__list {
+  .menu__list {
+    padding-left: 0;
+  }
+}
+
 .menu__link {
-  border-radius: 8px;
+  word-break: break-all;
```

Comment:
don't know if we should add this. i'm afraid of breaking something not properly

Reply:
Per design, sidebar width is fixed and should break to the next line if exceeds the length.

Design:
<img width="423" alt="image" src="https://user-images.githubusercontent.com/36393111/174735289-9850f126-e5bd-428d-911a-2cefa36ed536.png">

If not set to break all:
<img width="328" alt="image" src="https://user-images.githubusercontent.com/36393111/174735202-27eb5e10-8836-47c8-b0fc-feb9c7aebb98.png">

Reply:
i'm ok with both. let's confirm with design

Reply:
checked